astrology.lic

```diff
@@ -276,15 +285,30 @@ class Astrology
     fix_standing
   end
 
-  def train_astrology
-    check_ripples
-    check_astral
-    check_weather
-    check_events(check_pools) if @predict_event
-    check_moons
-    check_heavens
+  def train_astrology(settings)
+    return unless settings
+    return unless settings.astrology_training
+
```

Comment:
That makes sense, though it would be temporary code right? This is for the case that they update but have not reloaded dependency